cmd/kor/crds.go

```diff
@@ -11,7 +11,7 @@ import (
 
 var crdCmd = &cobra.Command{
 	Use:     "customresourcedefinition",
-	Aliases: []string{"crd", "customresourcedefinitions"},
+	Aliases: []string{"crd", "crds", "customresourcedefinitions"},
```

Comment:
This should be matched with the multi.go file
`"crd", "customresourcedefinition", "customresourcedefinitions"`

Reply:
I see no other resource has plural alias for initials, I'll revert the change.